src/Writer/XlsxWriter.php

```diff
@@ -26,35 +26,17 @@
  */
 final class XlsxWriter implements TypedWriterInterface
 {
-    /**
-     * @var string
-     */
-    private $filename;
+    private string $filename;
 
-    /**
-     * @var bool
-     */
-    private $showHeaders;
+    private bool $showHeaders;
 
-    /**
-     * @var bool
-     */
-    private $showFilters;
+    private bool $showFilters;
 
-    /**
-     * @var Spreadsheet
-     */
-    private $spreadsheet;
+    private ?Spreadsheet $spreadsheet = null;
 
-    /**
-     * @var Worksheet
-     */
-    private $worksheet;
+    private ?Worksheet $worksheet = null;
```

Comment:
Is correct to change both properties to nullable?

Reply:
The property is not initialized in the constructor so far.
Phpstan is complaining:
```
Class Sonata\Exporter\Writer\XlsxWriter has an uninitialized property $spreadsheet. Give it default value or assign it in the constructor.
```
If you call `write()` without `open()` before, you'll end with an access to an uninitialized property.

Later we will need to add some null check and throw some exceptions.